src/qmckl_ao.org

```diff
@@ -74,13 +91,22 @@ MunitResult test_qmckl_ao() {
       - ~LDP~ >= $\max_i$ ~LMAX[i]~
 
 ***** Header
-      #+BEGIN_SRC C :tangle (org-entry-get nil "h" t) 
-qmckl_exit_code qmckl_ao_power(const qmckl_context context,
-                const int64_t n, 
-                const double *X, const int32_t *LMAX,
-                const double *P, const int64_t LDP);
-      #+END_SRC
-    
+
+#+CALL: generate_h(d=args,rettyp=get_value("Type"),fname=get_value("Name"))
```

Comment:
It should be possible to pipe the result to an elisp function to nicely indent the produced C code.

Reply:
Ok. I also think that the function name should be changed to the following:
1. ```generate_c_interface``` for the c interface
2. ```generate_f_interface``` for the fortran interface 

What do you think?